amazon-kinesis/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,73 @@
+= Amazon Kinesis Data Streams Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
```

Comment:
Anypoint Connector for Amazon Kinesis Data Streams (Amazon Kinesis Data Streams Connector) provides access to standard Amazon Kinesis Data Stream operations using Anypoint Platform.

Amazon Kinesis is a streaming solution built by Amazon AWS to collect and process large streams of data records in real time. 

To learn more, please see Amazon Kinesis' official documentation.